src/m365/spo/commands/page/page-header-set.ts

```diff
@@ -192,6 +202,8 @@ class SpoPageHeaderSetCommand extends SpoCommand {
             header = defaultPageHeader;
         }
 
+        canvasContent = pageData ? pageData.CanvasContent1 : "";
```

Comment:
Seems like we're missing a test for when `pageData` is not set:

<img width="471" alt="image" src="https://user-images.githubusercontent.com/11164679/102987490-aed73780-4512-11eb-965e-8c9f014ac9cc.png">

Would you mind adding the missing test before we merge it to ensure that all of the code works as intended?

Reply:
Done, test is in.

Reply:
Awesome! Thank you! We'll check it shortly